src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/CourseActivityBuilder.kt

```diff
@@ -186,28 +182,6 @@ class CourseActivityBuilder(
     )
   }
 
-  override fun courseAllocation(
```

Comment:
Removed allocations from the course activity DSL for reasons specified above - it caused a circular reference in the builders.